ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPagerManager.java

```diff
@@ -58,4 +63,10 @@ public Map getExportedCustomDirectEventTypeConstants() {
   public void addView(ReactViewPager parent, View child, int index) {
     parent.addViewToAdapter(child, index);
   }
+
+  @Override
+  protected void onAfterUpdateTransaction(ReactViewPager view) {
```

Comment:
Why is this needed? I'm not very familiar with the ViewPager API. Can you add a comment?